tests/cli/status/test_status_module.py

```diff
@@ -429,26 +429,3 @@ def test_get_session_apierror(self):
             'Most Recent BOS Session': self.bos_session,
             'Most Recent Image': self.img_name,
         })
-
-    def test_get_sessiontemplate_apierror(self):
-        """Test retrieving component boot status when no sessiontemplate found"""
-        self.mock_bos_client.get_session_template.side_effect = APIError
```

Comment:
You should also remove the setting of `self.mock_bos_client.get_session_template.return_value` in the `setUp` method on lines 336-338 above.

```
        self.mock_bos_client.get_session_template.return_value = {
            'name': self.bos_sessiontemplate
        }
```